packages/p2p/src/components/advertiser-page/__tests__/advertiser-page.spec.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import AdvertiserPage from '../advertiser-page.jsx';
+
+jest.mock('@sendbird/chat', () => ({
+    SendbirdChat: jest.fn().mockReturnValue({}),
+}));
+
+jest.mock('@sendbird/chat/groupChannel', () => ({
+    SendbirdChat: jest.fn().mockReturnValue({}),
+}));
+
+jest.mock('@sendbird/chat/message', () => ({
+    SendbirdChat: jest.fn().mockReturnValue({}),
+}));
```

Comment:
@jim-deriv

Reply:
this issue will be fixed when this PR is merged: https://github.com/binary-com/deriv-app/pull/9569